generate_qt_resources.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Script to generate Qt Resource files for easy inclusion of qml-material into other projects.
+# The script generates four resource files:
+#   -Material.qrc:      The main qml-material components;
+#   -QtQuick.qrc:       The Material Design style required by qml-material's components;
+#   -FontAwesome.qrc:   The Font Awesome files
+#   -FontRoboto.qrc:    The Roboto font files (note: these are big!)
+
+
+cd modules
+
+function genereate_resource_file() {
```

Comment:
svg Material icons should also be added to a separate qrc

2016-01-20 17:32 GMT+01:00 Marco Piccolino <marco.a.piccolino@gmail.com>:

> how about using MaterialQtQuick.qrc instead of QtQuick.qrc? that should
> prevent confusion.
>
> 2016-01-20 16:38 GMT+01:00 Michael Spencer <notifications@github.com>:
>
>> In generate_qt_resources.sh
>> <https://github.com/papyros/qml-material/pull/375#discussion_r50269809>:
>>
>> > @@ -0,0 +1,48 @@
>> > +#!/bin/bash
>> > +#
>> > +# Script to generate Qt Resource files for easy inclusion of qml-material into other projects.
>> > +# The script generates four resource files:
>> > +#   -Material.qrc:      The main qml-material components;
>> > +#   -QtQuick.qrc:       The Material Design style required by qml-material's components;
>> > +#   -FontAwesome.qrc:   The Font Awesome files
>> > +#   -FontRoboto.qrc:    The Roboto font files (note: these are big!)
>> > +
>> > +
>> > +cd modules
>> > +
>> > +function genereate_resource_file() {
>>
>> Typo here. Should be generate_resource_file()
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/papyros/qml-material/pull/375/files#r50269809>.
>>
>
>